docs/user-guide/user-guide_en_EN.md

```diff
@@ -0,0 +1,121 @@
+# User's manual
+
+- [User's Manual](# User's Manual)
```

Comment:
Markdown rendered incorrectly in github page: 
https://github.com/sulaimangari/open-simulator/blob/added-en-guide/docs/user-guide/user-guide_en_EN.md